tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/ViewSubmittedPackageSD.java

```diff
@@ -69,16 +69,27 @@ public void verifyDocumentDownload() throws InterruptedException {
         Assert.assertEquals(0, downloadedFile.length());
     }
 
-    @And("comments are available in Filing Comments tab")
+    @Then("comments are available in Filing Comments tab")
     public void verifyFilingComments() {
         packageReviewPage.clickFilingCommentsTab();
         Assert.assertTrue(packageReviewPage.verifyFilingCommentsIsDisplayed());
 
     }
 
-    @And("payment status information is correct")
+    @Then("payment status information is correct")
     public void verifyPaymentStatus() {
         packageReviewPage.clickPaymentStatusTab();
         Assert.assertTrue(packageReviewPage.verifyPaymentStatusIsDisplayed());
     }
+
+    @Then("user can navigate to CSO and return to Efiling hub")
+    public void verifyUserCanNavigateToCsoAndReturnToEfilingHub() {

```

Comment:
should the url slug "wherearemypackage" be a parameter ? is it not a good candidate for scenarios ? what if you were to run the test against a running environment ?

Reply:
Thanks @alexjoybc. intention was to add environment specific assertions for dev as we progress enabling tests for dev. Will parameterize in scenarios. Thanks!